docs/index.asciidoc

```diff
@@ -130,6 +130,7 @@ Content type
 If not specified, this defaults to the following:
 
 * if format is "json", "application/json"
+* if format is "json_batch", "application/json". Each Logstash batch of events will be concatenated into a single array and sent in one request.
```

Comment:
there's a missing reference to the format config option:

```
| <<plugins-{type}s-{plugin}-format>> |<<string,string>>, one of `["json", "form", "message"]`|No
```

that should read

```
| <<plugins-{type}s-{plugin}-format>> |<<string,string>>, one of `["json", "json_batch", "form", "message"]`|No
```

Reply:
same for the extended `format` option description:

```

[id="plugins-{type}s-{plugin}-format"]
===== `format` 

  * Value can be any of: `json`, `form`, `message`
  * Default value is `"json"`
```

This should include the json_batch